assets/stylesheets/kitten/molecules/form/_form-composer.scss

```diff
@@ -0,0 +1,45 @@
+/// Setup a form composer element.
+///
+/// This mixin will setup an element to like a form composer
```

Comment:
Je proposerais la phrase suivante `This mixin will setup an element that places its children side by side`.